datafusion/tests/sql/expr.rs

```diff
@@ -714,17 +714,36 @@ async fn in_list_array() -> Result<()> {
 
 #[tokio::test]
 async fn test_extract_date_part() -> Result<()> {
-    test_expression!("date_part('hour', CAST('2020-01-01' AS DATE))", "0");
-    test_expression!("EXTRACT(HOUR FROM CAST('2020-01-01' AS DATE))", "0");
-    test_expression!(
-        "EXTRACT(HOUR FROM to_timestamp('2020-09-08T12:00:00+00:00'))",
-        "12"
-    );
     test_expression!("date_part('YEAR', CAST('2000-01-01' AS DATE))", "2000");
     test_expression!(
         "EXTRACT(year FROM to_timestamp('2020-09-08T12:00:00+00:00'))",
         "2020"
     );
+    test_expression!("date_part('MONTH', CAST('2000-01-01' AS DATE))", "1");
+    test_expression!(
+        "EXTRACT(month FROM to_timestamp('2020-09-08T12:00:00+00:00'))",
+        "9"
+    );
+    test_expression!("date_part('WEEK', CAST('2003-01-01' AS DATE))", "1");
+
+    //TODO Creating logical plan for 'SELECT EXTRACT(WEEK FROM to_timestamp('2020-09-08T12:00:00+00:00'))'
```

Comment:
Many keywords got added in  https://github.com/sqlparser-rs/sqlparser-rs/pull/427 from @mobuchowski (👋 ) but sadly it seems to be missing `week` 😭

Reply:
I didn't meant to make you feel bad @mobuchowski  -- mostly I was trying to give you props for adding stuff to sqlparser ;)

Reply:
Don't worry, that's not how I received it :)

Reply:
@Ted-Jiang Maybe you can file a pr to fix this.

Reply:
@liukun4515 @liukun4515  @mobuchowski   i would like to add this ,  it's a good chance to start familiar `sqlparser`
😊

Reply:
https://github.com/sqlparser-rs/sqlparser-rs/pull/436